ts/RailroadMap.ts

```diff
@@ -987,7 +993,8 @@ export class RailroadMap {
         if (frameText) {
             const yaw = normalizeAngle(frame.rotation.yaw);
             const flip = (yaw > -90) && (yaw < 90);
```

Comment:
nit: given this variable is named `flip`, I think it should be updated instead of adding a new variable. You can also simplify the boolean logic with `===` here.

```suggestion
            const flip = ((yaw > -90) && (yaw < 90)) === this.inverted;
```